cleanco/clean.py

```diff
@@ -16,21 +16,37 @@
 from collections import OrderedDict
 import re
 import unicodedata
-from .termdata import terms_by_type, terms_by_country
+from iso20275 import ElfEntry
 
 
 tail_removal_rexp = re.compile(r"[^\.\w]+$", flags=re.UNICODE)
 
 
-def get_terms():
+def get_unique_terms():
     "retrieve all unique terms from termdata definitions"
-    ts = functools.reduce(operator.iconcat, terms_by_type.values(), [])
-    cs = functools.reduce(operator.iconcat, terms_by_country.values(), [])
-    return set(ts + cs)
+    terms = []
+
+    for elf_code, values in Elf.items():
+        entity_codes = Elf[elf_code][0].local_abbreviations
+        if ";" in entity_codes:
+            split = entity_codes.split(';')
+            for split_item in split:
+                terms.append(split_item)
+        else:
+            terms.append(entity_codes)
+
+    terms = filter(None, terms)
+    terms = list(filter(None, terms))
```

Comment:
Why are we filtering a second time hear? Also, I think it might be clearer to use `all(terms)`  rather than filter with None as filter function?